esmf_utils/OutputInfo.F90

```diff
@@ -123,37 +130,47 @@ function get_vertical_dim_spec_names_bundle_info(info, rc) result(names)
       integer, optional, intent(out) :: rc
       integer :: status
       integer :: i
-      character(len=:), allocatable :: name
+      character(len=ESMF_MAXSTR) :: spec_name
 
+      spec_name = EMPTY_STRING
       names = StringVector()
       do i=1, size(info)
-         name = get_vertical_dim_spec_info(info(i), _RC)
-         if(find_index(names, name) == 0) call names%push_back(name)
+         spec_name = get_vertical_dim_spec_info(info(i), _RC)
+         _ASSERT(spec_name /= EMPTY_STRING, 'No vertical dim spec found.')
+         if(find_index(names, spec_name) == 0) call names%push_back(trim(spec_name))
       end do
       _RETURN(_SUCCESS)
 
    end function get_vertical_dim_spec_names_bundle_info
 
    function get_vertical_dim_spec_name_field(field, rc) result(spec_name)
-      character(len=:), allocatable :: spec_name
+      character(len=ESMF_MAXSTR) :: spec_name
```

Comment:
```suggestion
      character(len=:), allocatable :: spec_name
```

Reply:
I wasn't sure about the performance impact of using allocatable instead of fixed-length. I'll take a look.